pages/en/lb3/Authentication-authorization-and-permissions.md

```diff
@@ -68,55 +68,88 @@ module.exports = function enableAuthentication(server) {
 
 ### Preparing access control models
 
-Make sure the `User` model, (and possibly the `AccessToken` model) is configured appropriately according to your set-up.
+You must configure the `User` model(s), (and possibly the `AccessToken` model) according to your set-up.
 
-Normally you should have already implemented at least one custom user model, extending the built-in `User` model, as described in [Using built-in models](Using-built-in-models.html#user-model).
+The best practice is to implement at least one custom user model extending the built-in `User` model instead of using the built-in `User` model as-is, as described in [Using built-in models](Using-built-in-models.html#user-model).
 
```

Comment:
i just wanted here to avoid the repetition
1. **normally** you ..
2. usually you ..

and tried to be a bit more directive so do get developers' confidence